charts/fleet-agent/values.yaml

```diff
@@ -31,6 +31,19 @@ internal:
   systemNamespace: fleet-system
   managedReleaseName: fleet-agent
 
+# The nodeSelector and tolerations for the agent deployment
+nodeSelector: {}
```

Comment:
duplicated keys?

Reply:
I went back and forth on how to do this. Essentially, we need separate `nodeSelector` and `tolerations` settings for the agent because the rancher/kubectl container does not have Windows manifests. We might want to have a different way to convey this though. Perhaps...
```yaml
fleetAgent:
  nodeSelector:
  tolerations:
kubectl:
  nodeSelector:
  tolerations:
```

Reply:
Going with the above design.